samples/CppArExample/CppArExample.h

```diff
@@ -34,6 +34,7 @@ class CppArExample : public QObject
   Q_PROPERTY(Esri::ArcGISRuntime::Toolkit::ArcGISArView* arcGISArView READ arcGISArView WRITE setArcGISArView
              NOTIFY arcGISArViewChanged)
   Q_PROPERTY(Esri::ArcGISRuntime::SceneQuickView* sceneView READ sceneView WRITE setSceneView NOTIFY sceneViewChanged)
+  Q_PROPERTY(bool screenToLocationMode MEMBER m_screenToLocationMode NOTIFY screenToLocationModeChanged)
```

Comment:
When using `MEMBER` and you change `m_screenToLocationMode` in your C++ code, you must manually emit `screenToLocationModeChanged()`.

I don't see that happening anywhere, so not sure if that's needed or not.

Reply:
`m_screenToLocationMode` is not changed in C++, this is the reason I used `MEMBER` this time. But we don't use `MEMBER` usually, then I can change to setter/getter if you prefer.

Reply:
No, this is fine. In this case `MEMBER` is a lot easier and less code.